client/client-multi/client-c/src/kaa/platform-impl/common/key_utils.c

```diff
@@ -34,8 +34,8 @@
 
 #include <kaa_rsa_key_gen.h>
 
-#define KAA_PRIVATE_KEY_STORAGE "private.key"
-#define KAA_PUBLIC_KEY_STORAGE "public.key"
+#define KAA_PRIVATE_KEY_STORAGE "key.private"
+#define KAA_PUBLIC_KEY_STORAGE "key.public"
```

Comment:
The Java SDK and C++ SDK uses these names. Let's accept this and open the Jira tickets to change names in next version.